hack/build-antrea-linux-all.sh

```diff
@@ -155,14 +155,21 @@ cd build/images/ovs
 cd -
 
 cd build/images/base
-./build.sh $ARGS
+ARGS1="$ARGS --type controller"
```

Comment:
Instead of Arg1,2,3, prefer like CONTROLLER_ARGS="$ARGS --type controller"
AGENT_ARGS="$ARGS --type agent"
UNIFIED_ARGS="$ARGS --type unified"